python/cudf/cudf/tests/test_udf_masked_ops.py

```diff
@@ -802,6 +802,42 @@ def func(x):
     run_masked_udf_series(func, data, check_dtype=False)
 
 
+@pytest.mark.parametrize(
+    "data",
+    [
+        np.array(
+            [0, 1, -1, 0, np.iinfo("int64").min, np.iinfo("int64").max],
+            dtype="int64",
+        ),
+        np.array([0, 0, 1, np.iinfo("uint64").max], dtype="uint64"),
+        np.array(
+            [
+                0,
+                0.0,
+                -1.0,
+                1.5,
+                -1.5,
+                np.finfo("float64").min,
+                np.finfo("float64").max,
+                np.nan,
+                np.inf,
+                -np.inf,
+            ],
+            dtype="float64",
+        ),
+        [False, True, False, cudf.NA],
+    ],
+)
+def test_masked_udf_abs(data):
+    data = cudf.Series(data)
+    data[0] = cudf.NA
+
+    def func(x):
+        return abs(x)
```

Comment:
Does this also work if I, as a _user_  were to run `run_masked_udf_series(abs, data, check_dtype=False)`,  I suppose yes.

Reply:
Sadly not. This is an edge case that has come up a few times where we need to wrap builtin functions inside an object that has accessible bytecode. See more at https://github.com/rapidsai/cudf/issues/11595.